### DISCUSSION TITLE
Can't type the word 'paused'

### DISCUSSION THREAD

**Discussion by MoistToilet** [opensteno/plover#1346] [Q&A and Support] [answered]
In my dictionary it says PAFUD translates to 'paused', but when I actually go to type the word, it come out as 'payoffed'. I'm fairly new to Plover and have no idea why this is happening or how to fix it.

**Comment by benoit-pierre** [comment]
`PAFUD` is not valid steno.

**Comment by JorWat25** [comment, marked as answer]
It seems what's happening it that it's interpreting it as `PAUF` (payoff) + `-D`. So either 'paused' isn't actually in your dictionary, or you're entering the D separately. Check that you've not got the dictionary that came with v3, as that one doesn't seem to include `PAUFD`.

**Reply by MoistToilet** [reply]
When I look up 'paused' in my dictionary, it says the stroke is PAFUD, so I'm not sure why it won't work. I'm not sure which dictionary version I have. I don't mind having to go in there and edit things, but I don't understand why it won't work when I have it as a dictionary entry. Also, I'm pressing all the keys at once.

**Reply by benoit-pierre** [reply]
Because it's not valid: the `-F` key comes after `-U`, so you need to fix it to `PAUFD`.

**Reply by MoistToilet** [reply]
Oh, I understand what you mean now. Thank you so much!

**Comment by user202729** [comment]
Remark: Apart from the automatic suffix key handling thing, Plover could have automatic wrong-stroke detection on \<add translation>, \<load JSON dictionary>, \<dictionary editor>. For now I think only the plover_stroke library can detect wrong stroke, Plover can only normalize a stroke. (weird?)
